### PR TITLE
Feat/create a test suite to check for get single comment up vote controller

### PR DESCRIPTION
**Pull request by Alao-Abiodun**
**Before submitting your PR for review**

- Run `npm run lint` to find errors in code syntax/format
- Run `npm run lint:fix` to fix all auto-fixable errors in source code and auto-format with prettier
- Ensure you fix any linting errors displayed after running any of the above commands

**What does this PR do?**

Fixes #issue_number (replace issue_number with the issue number this PR solves)

Summarise the main tasks handled in the pull request

**description of Task to be completed?**

Describe (in detail) what the task completed in the pull request does as per the relevant task

**How should this be manually tested?**

Give bullet point instructions on how to setup the code and test it manually (Assume no prior experience on JS or code)

**Any background context you want to provide?**

Any pertinent information that should be considered

**What is the link to the issue on Github?**

[Issue #14](https://github.com/microapidev/comment-microapi/issues/14)

**Questions:**

If something is unclear or you want some questions to be addressed by your peers, mention them here

### CODE REVIEW

File: __test__/api/comments/updateSingleCommentUpVotes.test.js [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+        ownerId: ownerIdUpdate,
+      });
+
+    expect(removeRes.status).toEqual(200);
+    expect(removeRes.body.status).toEqual("success");
+    expect(removeRes.body.data).toEqual({
+      commentId: oldCommentWithUpVote.commentId,
+      numOfVotes: oldCommentWithUpVote.numOfVotes - 1,
+      numOfUpVotes: oldCommentWithUpVote.numOfUpVotes,
+      numOfDownVotes: oldCommentWithUpVote.numOfDownVotes - 1,
```
Comment (simply-alliv) [inline __test__/api/comments/updateSingleCommentUpVotes.test.js:199]: Since you are removing an upvote, shouldn't you be expecting the `numOfUpVotes` to be reduced by one. Not the `numOfDownVotes`.

File: __test__/api/comments/updateSingleCommentUpVotes.test.js [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+
+    // Run test matchers to verify that the comment vote has been removed in the database.
+    await CommentModel.findById(oldCommentWithUpVote.commentId).then(
+      (comment) => {
+        expect(comment).toBeTruthy();
+        expect(comment.upVotes.length).toEqual(
+          oldCommentWithUpVote.numOfUpVotes
+        );
+        expect(comment.downVotes.length).toEqual(
+          oldCommentWithUpVote.numOfDownVotes - 1
```
Comment (simply-alliv) [inline __test__/api/comments/updateSingleCommentUpVotes.test.js:210]: Same thing as the comment directly above.

File: __test__/api/comments/updateSingleCommentUpVotes.test.js [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+  it("Should return a 404 error when the commentId path parameter is invalid", async () => {
+    const url = `/v1/comments/4edd30e86762e0fb12000003/votes/upvote`;
+    const bearerToken = `bearer ${global.appToken}`;
+
+    const res = await request
+      .patch(url)
+      .set("Authorization", bearerToken)
+      .send({
+        ownerId: oldComment.ownerId,
+        content: "content",
```
Comment (simply-alliv) [inline __test__/api/comments/updateSingleCommentUpVotes.test.js:244]: There's no `content` property for this endpoint.

**Comment by simply-alliv** [comment]
So according to my assessment. You just copy and pasted and didn't critically read through the code and understand what it is really doing.

These are tests that are suppose to ensure that our service is never broken or buggy. It is important that the tests always test the right thing.

**Comment by simply-alliv** [comment]
But everything else is great. Well done.

File: __test__/api/comments/updateSingleCommentUpVotes.test.js [hunk 4]
```diff
@@ -233,7 +233,7 @@ describe("PATCH /comments/:commentId/votes/upvote", () => {
   });
 
   it("Should return a 404 error when the commentId path parameter is invalid", async () => {
-    const url = `/v1/comments/4edd30e86762e0fb12000003/votes/upvote`;
+    const url = `/v1/comments/4edd30e86762e0fb12000003`;
```
Comment (dave-ok) [inline __test__/api/comments/updateSingleCommentUpVotes.test.js:237]: Why did you change this? The endpoint was correct the first time around
